### ISSUE TITLE
GenConfig configurations should never force disable a test

### ISSUE THREAD

**Issue by bartlettroscoe** [trilinos/Trilinos#10870]
CC: @trilinos/framework

### Internal issues

* [TRILINOSHD-167](https://sems-atlassian-son.sandia.gov/jira/servicedesk/customer/portal/7/TRILINOSHD-167)

### Description

Looking at the file:

* `packages/framework/ini-files/config-specs.ini`

I see that `FORCE` is used with a bunch of test disables like:

```
opt-set-cmake-var Adelus_vector_random_MPI_1_DISABLE BOOL FORCE : ON
```

You **NEVER** want to force disable a test.  If you force disable it, the developer has no way to manually enable that test in order to fix it.

**Comment by bartlettroscoe** [comment]
This is addressed by PR #10871.  Putting in review.

**Comment by bartlettroscoe** [comment]
This was resolved with the merge of #10847.

Closing as complete.
